gov2/s3/GeneratePresignedURL/GeneratePresignedURLv2.go

```diff
@@ -0,0 +1,78 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX - License - Identifier: Apache - 2.0
```

Comment:
Not a huge deal, but the canonical form is "SPDX-License-Identifier: Apache-2.0" (without spaces), per https://spdx.dev/ids/

Reply:
VS Code (not) helpfully put spaces around the dashes in my code snippet. Fixed.